log/sequencer.go

```diff
@@ -166,8 +171,12 @@ func (s Sequencer) createRootSignature(ctx context.Context, root trillian.Signed
 // which will fail if the tx was committed. Should only do this if we can hide the details of
 // the underlying storage transactions and it doesn't create other problems.
 func (s Sequencer) SequenceBatch(ctx context.Context, limit int) (int, error) {
-	tx, err := s.logStorage.Begin(ctx)
+	logID, err := util.LogID(ctx)
```

Comment:
Yep, I didn't like this either TBH. I had a parameter at some point, but then it and the context's logID could differ, which meant more changes and so on.

I'll add a TODO.